### ISSUE TITLE
How to use different metrics for multi-output tasks?

### ISSUE THREAD

**Issue by wsstriving** [fchollet/keras#5851]
Hello, 
       I want to train a multitask network with 2 outputs, the first one with CE metric, and the second one with Cosine Similarity. I tried the following codes:
      model.compile(loss='categorical_crossentropy',  'cosine_proximity') , but it seems to me both tasks are optimized with both metrics, thus 4 losses are printed out(task1 ce, task1 cosine, task2 ce, task 2 cosine). I wonder how to let task 1 only be optimized by CE, and task 2 only be optimized by Cosine (task1 ce, task2 cosine)?
